src/2d/Sprite.cpp

```diff
@@ -9,17 +9,29 @@ namespace ari::en
 
 	static gfx::PosVertex2D s_SpritePosVertices[] =
 	{
-		{-1.0f, -1.0f},
-		{1.0f, -1.0f},
-		{1.0f, 1.0f},
-		{-1.0f, 1.0f},
+		{ 0.0f, 0.0f, 0.0f, 0.0f },
+		{ 1.0f, 0.0f, 1.0f, 0.0f },
+		{ 1.0f, 1.0f, 1.0f, 1.0f },
+		{ 0.0f, 1.0f, 0.0f, 1.0f },
```

Comment:
Maybe we need to change it later and set the position from -0.5 to 0.5

Reply:
Because of rotation if we change the position to -0.5 to 0.5 then it rotates from center, not the top left.

Reply:
yeah, Got it!